metricbeat/docs/faq-unexpected-metrics.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[faq-unexpected-metrics]]
+=== {beatname_uc} collects system metrics for interfaces you didn't configure
```

Comment:
Is there a reason this is limited to the system module? All but a few modules have a concept of a "default metricset"

Reply:
@fearful-symmetry  yes...because the system modules is enabled by default, users are more likely to run into this problem with the system module. If they've explicitly enabled or disabled metricsets in a different module hopefully they won't run into this problem.